artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/components/addressSendMessage.js

```diff
@@ -42,6 +42,20 @@ var Artemis;
                         <div class="form-group">
                             <label>Durable </label>
                             <input id="durable" type="checkbox" ng-model="$ctrl.message.durable" value="true">
+                            <button type="button" class="btn btn-link jvm-title-popover"
+                                      uib-popover-template="'durable-info.html'" popover-placement="bottom-left"
+                                      popover-title="Durable" popover-trigger="'mouseenter'">
+                                <span class="pficon pficon-info"></span>
+                            </button>
+                        </div>
+                        <div class="form-group">
+                            <label>Create User ID </label>
+                            <input id="messageID" type="checkbox" ng-model="$ctrl.message.messageID" value="true">
```

Comment:
Label needs updated (line 52 if this isnt clear due to the existing comments on it)